crm/track_leads/lead_scoring.rst

```diff
@@ -2,11 +2,11 @@
 Assign leads based on scoring
 =============================
 
-With *Leads Scoring* you can automatically rank your leads based on
-selected criterias.
+With *Lead Scoring* you can automatically rank your leads based on
+selected criteria.
 
-For example you could score customers from your country higher or the
-ones that visited specific pages on your website.
+For example, you could score customers higher from your country higher or for
+visiting specific pages on your website.
 
 Configuration
 =============
```

Comment:
General comment: you should update screenshots whenever visual differences are introduced. It seems to be the case here in that particular case here i.e. Lead Scoring module has a description on Odoo now. Please check all other screenshots.
Also, all the images are missing [alt tags](https://www.odoo.com/documentation/user/14.0/contributing/documentation/content_guidelines.html#alt-tags), which should be added.
In addition, if you replace a file (and only if), think of renaming it. The new naming guidelines can be found [here](https://www.odoo.com/documentation/user/12.0/contributing/documentation/content_guidelines.html) (the instructions have been done on a v12 doc, but they haven't done the forward ports of this to v13 and v14, but they will be the new standard very soon)